tests/scripts/tmate-pod.yaml

```diff
@@ -0,0 +1,79 @@
+# Apply this manifest to CI clusters to allow debugging complex failures during CI runs
+#
+# Usage:
+#   kubectl apply -f tests/scripts/tmate-pod.yaml
+#   kubectl -n tmate wait --for=condition=ready -l app=tmate pod --timeout=300
+#   sleep 1 # just in case tmate hasn't output its web/ssh links yet
+#   kubectl -n tmate logs deploy/tmate
```

Comment:
Does this print the ssh connection string?

Reply:
Yes